tests/unit/converters/test_base_converter.py

```diff
@@ -117,6 +117,20 @@ def test_load_asym_output_data(converter: DummyConverter):
     assert data == {"foo": 1}
 
 
+def test_load_sc_output_data(converter: DummyConverter):
+    # Arrange
+    converter._parse_data.return_value = {"foo": 1}  # type: ignore
```

Comment:
Why do you need to pylint ignore?

Reply:
@mgovers because those functions dont have those attributes. The same ignoring is done for other functions. I see some way of adding specs exist via mock but not sure if thats correct here. Do you know a better way?

Reply:
nah you can leave it be